fdbserver/LogSystem.h

```diff
@@ -996,9 +996,13 @@ struct LogPushData : NonCopyable {
 		writtenLocations.clear();
 	}
 
-	// copy next_message_tags into given set
-	void saveTags(std::set<Tag>& writtenTags) {
-		writtenTags.insert(next_message_tags.begin(), next_message_tags.end());
+	// copy written_tags into given set
+	void saveTags(std::set<Tag>& writtenTags, bool filterPseudoTags = true) {
```

Comment:
I find the similarly named `written_tags` and `writtenTags` harder to read, renaming `writtenTags` to `filteredTags` or `returnedTagList`  would help. 

I think you could remove the parameter `filterPseudoTags` as the extra logic is not needed.

Reply:
Done.